### ISSUE TITLE
I tested jitsi-hammer on openfire and have some problems.

### ISSUE THREAD

**Issue by yue-fei** [jitsi/jitsi-hammer#4]
I use windows, and I build jitsi-hammer, and copy the lib->native->windows to the jre lib.

In openfire, I create a room name TestHammer, but when I start the jitsi-hammer, then I open chrome, I find nothing happened, just black,.

parameters XMPPdomain, XMPPhost, ivffile, videortpdump and so on, that they are all configured, else I found the log 'audio":{ "ssrc":-1 , "DownloadJitterMs":NaN , "DownloadPercentLoss":NaN .....'

Can you give me some tips?

**Comment by Haerezis** [comment]
Try to create a jitsi-meet conference (with at least 2 users) without Jitsi-Hammer first.
If it doesn't work, it means something is wrong with you Jitsi-Meet config or Openfire config.

If it works : you need to create a Jitsi-Meet conference with chrome BEFORE running Jitsi-Meet, because it's chrome/jitsi-meet that handle the conference.

**Comment by yue-fei** [comment]
Thank you very much @Haerezis 
Myy jitsi-meet seems has some wrong. When I refresh the page, it's show.

And I found hammer log was NaN, just like ''audio":{ "ssrc":-1 , "DownloadJitterMs":NaN , "DownloadPercentLoss":NaN .....''

I changed some code here, it's show.

![image](https://cloud.githubusercontent.com/assets/10253422/7131818/0d899626-e2b6-11e4-9e87-795e6b3a6fc3.png)

![image](https://cloud.githubusercontent.com/assets/10253422/7131823/13ed8c3e-e2b6-11e4-958a-3f7f12a371de.png)

![image](https://cloud.githubusercontent.com/assets/10253422/7131832/1ed85386-e2b6-11e4-9113-92371d4c1afd.png)

**Comment by Haerezis** [comment]
Yeah, some change have broken the stats recording, that's why you get gibberish data.
If you don't need them, you can disable like you did (but normally, there's an option for that...).
